integration/app/test-demo/src/containers/MeetingControls.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {
+  ControlBar,
+  AudioInputControl,
+  VideoInputControl,
+  AudioOutputControl,
+  ContentShareControl,
+} from 'amazon-chime-sdk-component-library-react';
+import MeetingLeaveControl from '../components/MeetingLeaveControl';
+
+const MeetingControls: React.FC = () => (
+  <ControlBar
+    layout='undocked-horizontal'
+    showLabels
+  >
+    <AudioInputControl />
+    <VideoInputControl />
+    <AudioOutputControl />
```

Comment:
just wonder do we need audio output here, does it change any attendee states?

Reply:
We don't need it for roster test. But since this is a `MeetingControls` container, I think it's good to keep it. How do you think?